src/GitlabCli/Git.psm1

```diff
@@ -15,7 +15,18 @@ function Get-LocalGitContext {
         Set-Location $Path
         if (Get-ChildItem -Filter '.git' -Hidden -Directory) {
             $Context.Repo = $($(git config --get remote.origin.url) -split ':' | Select-Object -Last 1).Replace('.git', '')
-            $Context.Branch = git rev-parse --abbrev-ref HEAD
+            
+            $branchOrSha = git status | select-string "^HEAD detached at (?<sha>.{8})`|^On branch (?<branch>.*)"
```

Comment:
nitpick: casing

I hate to comment on trivial things like casing, but trying to keep things consistent.

Reply:
(specifically variables and cmdlets)

Reply:
Consistencized (not a word but sounds neat)